integration/code/connecting_to_your_data/filesystem/pandas_python_example.py

```diff
@@ -0,0 +1,47 @@
+import great_expectations as ge
+from great_expectations.core.batch import RuntimeBatchRequest
+
+context = ge.get_context()
+
+datasource_config = {
+    "name": "taxi_datasource_with_runtime_data_connector",
+    "class_name": "Datasource",
+    "module_name": "great_expectations.datasource",
+    "execution_engine": {
+        "module_name": "great_expectations.execution_engine",
+        "class_name": "PandasExecutionEngine",
+    },
+    "data_connectors": {
+        "default_runtime_data_connector_name": {
+            "class_name": "RuntimeDataConnector",
+            "batch_identifiers": ["default_identifier_name"],
+        }
+    },
+}
+
```

Comment:
Will need to add the `InferredAssetDataConnector` here and in the `pandas_example.py` file too. What to do with the `base_directory` is an open question :/